src/cascadia/TerminalApp/ColorScheme.cpp

```diff
@@ -138,32 +141,37 @@ void ColorScheme::LayerJson(const Json::Value& json)
     }
 }
 
-std::wstring_view ColorScheme::GetName() const noexcept
+winrt::hstring ColorScheme::Name() const noexcept
 {
-    return { _schemeName };
+    return { _schemeName.c_str() };
 }
 
-std::array<til::color, COLOR_TABLE_SIZE>& ColorScheme::GetTable() noexcept
+winrt::com_array<Color> ColorScheme::Table() noexcept
 {
-    return _table;
+    winrt::com_array<Color> result = winrt::com_array<Color>(COLOR_TABLE_SIZE);

```

Comment:
unexpected/unnecessary constructor call

```suggestion
    winrt::com_array<Color> result{ COLOR_TABLE_SIZE };
```